_config.yml

```diff
@@ -92,7 +93,7 @@ authors:
   dave:
     name: Dave Goldberg
     display_name: Dave Goldberg
-    email: ""
+    email: "dave.golberg@capitalone.com"
```

Comment:
The author of this guest blog post: https://blog.apiary.io/2015/05/03/The-Role-of-the-API-Designer